src/api.py

```diff
@@ -0,0 +1,28 @@
+import requests 
+import hashlib
+
+
+class response:
+    def __init__(self, filename, type_of_data, size, url, checksum):
+        self.filename = filename
+        self.type_of_data = type_of_data
+        self.size=size
+        self.url=url
+        self.checksum=checksum
+
+
+def fetch(url):
+    try:
```

Comment:
Do not wrap your code in a try/except. If there are errors let them bubble up

Reply:
Hi, then how should I reslove this https://github.com/nexB/fetchcode/pull/4#discussion_r333590130